crowbar_framework/app/models/api/upgrade.rb

```diff
@@ -348,9 +348,10 @@ def openstackbackup
         ::Crowbar::UpgradeStatus.new.start_step(:backup_openstack)
 
         crowbar_lib_dir = "/var/lib/crowbar"
-        dump_path = "#{crowbar_lib_dir}/upgrade/6-to-7-openstack_dump.sql"
+        dump_path = "#{crowbar_lib_dir}/upgrade/6-to-7-openstack_dump.sql.tar.gz"
```

Comment:
Shouldn't we put it under `/var/lib/crowbar/backup/` , just like the crowbar (admin) backup?